packages/app/src/cli/models/app/app.ts

```diff
@@ -231,6 +232,12 @@ export class App implements AppInterface {
     )
   }
 
+  get draftExtensions() {
```

Comment:
better to name this `draftableExtensions`, what do you think? `draftExtensions` could be confusing and meaning we are returning drafts